end_users/README.md

```diff
@@ -22,3 +28,79 @@ For more on these options and what they mean, read the [documentation on merge m
 In order for BonnyCI to function correctly, please only allow merge commits. From your repository's home page, navigate to the `Settings` tab. Scroll down to the section labeled `Merge button`, and ensure that it matches the following:
 
 ![Correct Merge Button Configuration](../misc/images/mergebutton.png)
+
+## Bug Reports
+
+If you encounter an issue which you think is a bug, please submit it to the [projman issue tracker](https://github.com/BonnyCI/projman/issues). Make sure that a similar bug has not already been filed, and post on that bug's issue conversation if it has.  The bug report will be reviewed and may be forwarded to an appropriate BonnyCI subrepository.
+
+### Bug Report Requirements
+
+* A descriptive title. Use keywords so others can find your bug to avoid duplicates and improve its visibility.
+
+* Limited to one bug. If you have another bug to report, please raise a separate issue.
+
+* Links to your project's:
+  * relevant BonnyCI logs
+  * BonnyCI configuration file, `.bonnyci`
+  * Any other miscellaneous scripts that BonnyCI might be kicking off.
+
+* The expected result or output that BonnyCI will produce.
+
+* The actual result or output that BonnyCI is currently producing.
+
+* The following details about your environment:
+  * BonnyCI version or latest SHA if you are running from source.
+  * Any tools you may be installing that are relevant to the issue (ex: if BonnyCI is running a linter, include a link to this linter).
+  * The primary language(s) of your project.
+
+* The red `bug` label.
+
+* Please do not add a milestone label. This will be added by the BonnyCI Admin team.
+
+The BonnyCI admin team will be alerted as soon as the bug report is summitted, and a team member will address it as soon as possible. Further clarification or details may be required depending on the situation.
```

Comment:
We should be careful with the wording here. We don't want to imply a SAL that does not exist. I would suggest:

```markdown
The BonnyCI admin team will be alerted after the bug report is submitted, and a team member will respond. Further clarification or details may be required depending on the situation.
```